src/AlixBa/Addic7edSubtitles/Helpers/Episode.php

```diff
@@ -64,6 +64,18 @@ public function __construct($episodeFilename)
      *
      * @return string
      */
+    public static function capitalizeShowName($showName)

```

Comment:
Nope. I just tried to follow existing code, where sanitizeShowName was declared as static.

Also, since the code is not touching the object, i thought static were fine here.